gensim/parsing/preprocessing.py

```diff
@@ -40,6 +40,26 @@
 
 
 def remove_stopwords(s):
+    """Takes string, removes all words those are among stopwords.
```

Comment:
Can you start the docstring on its own line? Not continue after `"""`, it's harder to read.

Reply:
It's [numpy-style convention](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard).

Reply:
Hmm, ok.